assembly/fabric8-stacks/src/main/resources/stacks.json

```diff
@@ -217,6 +217,18 @@
           "name": "Get OpenShift Console URL",
           "type": "",
           "attributes": {}
+        },
+        {
+          "commandLine": "cd /projects/formatter && yarn",
+          "name": "Build Plugin",
```

Comment:
@l0rd Would it better to name these commands `Build Formatter Plugin` and `Publish Formatter Plugin` since information about formatter is hard-coded?

Reply:
Can you automatically get the project folder name (you can assume that there is just one folder in /projects/). That would allow to use the same command for all plugins.

Reply:
Sure, `cd /projects/*` should work fine in this case I suppose. I will try

Reply:
I don't think it will work. You may check if there is a `package.json` in current folder and run yarn:
`[ -f ./package.json ] && yarn || echo ERROR: no package.json found in current folder $(pwd)`